### PR TITLE
libhandy1: update to 1.6.1

### PR DESCRIPTION
**Pull request by oreo639**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/libhandy1/template [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+version=1.6.1
 revision=1
 wrksrc="libhandy-${version}"
 build_style=meson
 build_helper="gir"
 configure_args="-Dexamples=true -Dtests=true $(vopt_bool gtk_doc gtk_doc)
  $(vopt_bool gir vapi) -Dglade_catalog=$(vopt_if glade enabled disabled)
  -Dintrospection=$(vopt_if gir enabled disabled)"
-hostmakedepends="glib-devel pkg-config $(vopt_if gtk_doc gtk-doc) $(vopt_if gir vala-devel)"
+hostmakedepends="glib-devel pkg-config $(vopt_if gtk_doc 'gtk-doc gi-docgen') $(vopt_if gir vala-devel)"
```
Comment (paper42) [inline srcpkgs/libhandy1/template:11]:
```suggestion
hostmakedepends="glib-devel pkg-config $(vopt_if gtk_doc gi-docgen) $(vopt_if gir vala-devel)"
```
or is there a reason why you also added gtk-doc?

Comment (paper42) [inline srcpkgs/libhandy1/template:11]:
```suggestion
hostmakedepends="glib-devel pkg-config gettext $(vopt_if gtk_doc 'gtk-doc gi-docgen') $(vopt_if gir vala-devel)"
```

Comment (oreo639) [reply srcpkgs/libhandy1/template]: Thanks, tbh I can't remember why I thought it was necessary to leave it in.

**Comment by tibequadorian** [comment]
The cross build switch for docs may be no longer necessary

**Pull request merged**